tailcfg/tailcfg.go

```diff
@@ -2286,6 +2286,11 @@ const (
 	// NodeAttrSSHBehaviorV1 forces SSH to use the V1 behavior (no su, run SFTP in-process)
 	// Added 2024-05-29 in Tailscale version 1.68.
 	NodeAttrSSHBehaviorV1 NodeCapability = "ssh-behavior-v1"
+
+	// NodeAttrSplitDNSForBattery configures the DNS resolver to adopt a split DNS configuration
+	// in more cases, offloading some queries to the OS. It is currently only sent from control
+	// to iOS devices.
+	NodeAttrSplitDNSForBattery NodeCapability = "split-dns-for-battery"
```

Comment:
I'd like to remove the "for-battery" from the name and use the saved characters to say more what it technically does. Can the docs here elaborate on "in more cases"?

Reply:
I changed this to be a negative attr (so that we don't have to send it down until we decide to disable this), and elaborated in the comments (not easy to explain this in few sentences)